app/styles/_variables.scss

```diff
@@ -24,6 +24,7 @@ $ilios-orange: #c60;
 $background-grey: #fbfbfb;
 $toggle-background-grey: #eceeef;
 $light-grey: #fefefe;
+$light-grey2: #d3d3d3;
```

Comment:
This would usually be the right place for this, but there are so many greys here already I think this should go into learner-group-members.scss as the variable $learnerGroupGrey